python/test/unit/fem/test_function.py

```diff
@@ -173,7 +173,7 @@ def f(x):
     assert round(w.x.norm(la.Norm.l1) - 6 * num_vertices, 7) == 0
 
 
-@pytest.mark.parametrize("types", [
+@ pytest.mark.parametrize("types", [
     # (np.float32, "float"),  # Fails on Redhat CI, needs further investigation
```

Comment:
We should file an issue on this, also noticed it.

Reply:
Re-enabled, works.